src/Microsoft.Azure.WebJobs.Extensions.EventHubs/EventHubsWebJobsStartup.cs

```diff
@@ -9,6 +10,7 @@
 
 namespace Microsoft.Azure.WebJobs.EventHubs
 {
+    [Extension("EventHubs")]
```

Comment:
Actually, I think it might make sense at this point to generalize this attribute to WebJobsStartupAliasAttribute

Reply:
If this becomes a requirement, we might just want to add the name to the existing `WebJobsStartupAttribute`. The issue with the dependency on internal types isn't that pressing as we could just rely on abstractions. My concern is with some of the downsides we've discussed.